bootbox.js

```diff
@@ -317,9 +319,15 @@
 
   exports.confirm = function() {
     var options;
+    var invertButtons = defaults.invertButtons;
+    var dialog;
 
     options = mergeDialogOptions("confirm", ["cancel", "confirm"], ["message", "callback"], arguments);
```

Comment:
Fair point :). I haven't had proper dig back through my own code in a while, sorry. Looking at it again, sanitize(options) is always called and it's this which decides button classes; so this method would know about options.invertButtons too. Maybe I'm overreacting to the DOM 'hack' - interested in getting a bit more feedback first (i.e. from @tarlepp) - sorry to keep you waiting even longer.